cypress/integration/brewery/TableParameters-tests.spec.js

```diff
@@ -260,6 +261,40 @@ describe('Table parameters standard operations', () => {
     BreweryParameters.getCustomersTableCell('height', 3).should('have.text', '2.01');
   });
 
+  it('can import a CSV file, edit it, import a new CSV file and override the former one, update and launch', () => {
+    const scenarioName = forgeScenarioName();
+    scenarioNamesToDelete.push(scenarioName);
+    Scenarios.createScenario(scenarioName, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE);
+    BreweryParameters.switchToCustomersTab();
+    ScenarioParameters.edit();
+    BreweryParameters.importCustomersTableData(CSV_VALID_FILE_PATH);
+    BreweryParameters.getCustomersTableRows().should('have.length', 4);
+    BreweryParameters.getCustomersTableCell('name', 0).should('have.text', 'Bob');
+    BreweryParameters.getCustomersTableCell('age', 0).should('have.text', '10');
+    BreweryParameters.getCustomersTableCell('favoriteDrink', 0).should('have.text', 'AppleJuice');
+
+    BreweryParameters.editCustomersTableStringCell('age', 0, '22').should('have.text', '22');
+    BreweryParameters.editCustomersTableStringCell('canDrinkAlcohol', 0, 'true').should('have.text', 'true');
```

Comment:
If we edit `canDrinkAlcohol`, can we check its previous value before the edition ?